### DISCUSSION TITLE
Images of my all projects are not showing or corrupt.

### DISCUSSION THREAD

**Discussion by sahilatahar** [community/community#42991] [New to GitHub] [answered]
Hello Sir, I am cofuse that what happen. I upload my project through git and also by uploading files but my image and screenshot images not showing ,not downloading.please help me to fix it.Thank you.

**Comment by diva-21** [comment, marked as answer]
Can you please keep a screenshot of that issue to understand it better @Sahil-Atahar

**Reply by sahilatahar** [reply]
THERE WAS MY INTERNET ISSUE

**Comment by rodri613** [comment]
### rodrigo mancilla bautista
